sympy/plotting/plot.py

```diff
@@ -227,16 +227,21 @@ def append(self, arg):
         second plot's first series object to the first, use the
         ``append`` method, like so:
 
-        >>> from sympy import symbols
-        >>> from sympy.plotting import plot
-        >>> x = symbols('x')
-        >>> p1 = plot(x*x)
-        >>> p2 = plot(x)
-        >>> p1.append(p2[0])
-        >>> p1
-        Plot object containing:
-        [0]: cartesian line: x**2 for x over (-10.0, 10.0)
-        [1]: cartesian line: x for x over (-10.0, 10.0)
+        .. plot::
+           :context:
+           :format: doctest
+           :include-source: True
+
+           >>> from sympy import symbols
+           >>> from sympy.plotting import plot
+           >>> x = symbols('x')
+           >>> p1 = plot(x*x, show = False)  # doctest: +SKIP
+           >>> p2 = plot(x, show = False)  # doctest: +SKIP
```

Comment:
No spaces around the `=` between show and false. See PEP8 for guidelines on style.

Why do these two lines need to be skipped?